src/app/src/css/about.css

```diff
@@ -0,0 +1,19 @@
+.about {
+    padding-top: 1rem;
+}
+
+.about > h2 {
+    text-align: center;
+}
+
+.about__slide {
+    height: 80vh;
+    padding: 1rem;
+}
+
+/* nuka-carousel marks previous slides as aria-hidden
+   This means we need to override our aria-hidden style so we
+   can still see slivers of the surrounding slides */
+.about__slide[aria-hidden="true"] {
```

Comment:
The slides being hidden is actually a cause of CSS in `navbar.css` that hides all elements where `aria-hidden="true"`, https://github.com/azavea/pwd-fishway/blob/develop/src/app/src/css/navbar.css#L8

I also couldn't figure out nested CSS for whatever reason which would prevent this run-in 🤔 We can update the comment in this `about.css` and leave this CSS rule or adjust the `navbar.css` while getting rid of this CSS rule.

Reply:
Went with option 2 and scoped the selector in `navbar.css` to only apply to the tab panes.